css/main.css

```diff
@@ -71,17 +76,17 @@ main a {
 }
 
 footer {
-    background-color: black;
+    background-color: #464646;
     padding: 1rem 1rem 2rem;
     width: 100%;
 }
 
 footer > div {
-    background-color: black;
+    background-color: #464646;
     color: white;
     display: flex;
     flex-direction: row;
-    font-family: Georgia, serif;
+    font-family: 'Source Sans Pro', sans-serif;
```

Comment:
our second font, and font for instructions should be 'Avenir'

Reply:
Avenir ins't a google font https://fonts.google.com/?query=Avenir

Reply:
You can put Avenir as 1st choice, then keep Source Sans Pro. It works even without adding google font link